components/style/values/specified/grid.rs

```diff
@@ -46,15 +46,15 @@ impl ToCss for GridLine {
         }
 
         if self.is_span {
-            try!(dest.write_str("span"));
+            dest.write_str("span")?;
         }
 
         if let Some(i) = self.integer {
```

Comment:
This one needs to be `Integer` (assuming it's the specified value), in order to correctly handle `calc()`.